connect/runtime/src/main/java/org/apache/kafka/connect/runtime/SinkConnectorConfig.java

```diff
@@ -96,6 +101,25 @@ public static void validate(Map<String, String> props) {
             throw new ConfigException("Must configure one of " +
                 SinkTask.TOPICS_CONFIG + " or " + SinkTask.TOPICS_REGEX_CONFIG);
         }
+
+        if (hasDlqTopicConfig) {
+            String dlqTopic = props.get(DLQ_TOPIC_NAME_CONFIG).trim();
+            if (hasTopicsConfig) {
+                List<String> topics = parseTopicsList(props);
+                if (topics.contains(dlqTopic)) {
+                    throw new ConfigException(DLQ_TOPIC_NAME_CONFIG + " has a topic name which is already in " +
```

Comment:
Should we log the topic name for this exception? For example, ```has a topic name (xxx) which```

Reply:
There will be just one topic in the DQL topic config. We could add it but I'm not sure if it's explicitly necessary.